python/tank/hook.py

```diff
@@ -631,17 +656,33 @@ def create_hook_instance(hook_paths, parent):
 
         3. /tmp/c.py is finally loaded in, get_hook_baseclass() now returns HookB.
 
-        4. HookC class is instantiated and method method_name is executed.
+        4. HookC class is returned.
```

Comment:
in step 1, can we clarify? 

"get_hook_baseclass() will return Hook at this point" --> "get_hook_baseclass() will return Hook at this point or a custom base class (deriving from hook) if this is passed in via the base_class param"?

Reply:
done.